crates/ott-balancer/Cargo.toml

```diff
@@ -35,6 +35,7 @@ route-recognizer = "0.3.1"
 once_cell.workspace = true
 pin-project.workspace = true
 prometheus.workspace = true
+enum_dispatch = "0.3.13"
```

Comment:
This should be in the root `Cargo.toml` and referenced in here just like all the other dependencies here.